common/annotations/sr_Latn.xml

```diff
@@ -12,7 +12,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<script type="Latn"/>
 	</identity>
 	<annotations>
-		<annotation cp="{">leva vitičasta zagrada | vitičasta zagrada | zagrada</annotation>
+		<annotation cp="{">vitičasta zagrada | zagrada | leva vitičasta zagrada</annotation>
 		<annotation cp="{" type="tts">leva vitičasta zagrada</annotation>
```

Comment:
Why has the ordering changed from alphabetical to non-alphabetical? Same question for changes below in this file